dao/src/main/java/uk/gov/moj/sdt/dao/BulkSubmissionDao.java

```diff
@@ -94,24 +64,30 @@ public IBulkSubmission getBulkSubmission(final IBulkCustomer bulkCustomer, final
             criteriaBuilder.lower(root.get("customerReference")),
             customerReference.toLowerCase()
         );
+        Predicate sdtDatePredicate = createDatePredicate(criteriaBuilder, root, dataRetention);
+
         TypedQuery<BulkSubmission> typedQuery = getEntityManager().createQuery(criteriaQuery.select(root)
                                                                                    .where(
-                                                                                       sdtCustomerPredicate,
-                                                                                       customerReferencePredicate,
-                                                                                       createDatePredicate(criteriaBuilder, root, dataRetention)
+                                                                                       sdtCustomerPredicate
+                                                                                       ,customerReferencePredicate
+                                                                                       ,sdtDatePredicate
```

Comment:
Is having commas at the beginning of the line the recommended coding style here?  Just looks a little odd.  If you change it make the same change in the getBulkSubmissionBySdtRef() method.

Reply:
Helped with debugging, switching off parameters. Reverted.